website/content/docs/discovery/dns.mdx

```diff
@@ -189,13 +194,13 @@ whether you want to filter results based on a service tag:
 - No filtering on service tag:
 
    ```text
-  _<service>._tcp[.service][.<datacenter>].<domain>
+  _<service>._tcp[.service][.<datacenter-or-peer>].<domain>
```

Comment:
Intended to support in the future, but no support now since it's always parsed as dc